backend/src/db/factories.js

```diff
@@ -4,9 +4,16 @@ import slugify from 'slug'
 import { hashSync } from 'bcryptjs'
 import { Factory } from 'rosie'
 import { getDriver, getNeode } from './neo4j'
+import CONFIG from '../config/index.js'
 
 const neode = getNeode()
 
+const uniqueImageUrl = imageUrl => {
+  const newUrl = new URL(imageUrl, CONFIG.CLIENT_URI)
+  newUrl.search = `random=${uuid()}`
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-25T14:30:28Z" title="Tuesday, February 25th 2020, 3:30:28 pm +01:00">Feb 25, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -4,9 +4,16 @@ import slugify from 'slug'
 import { hashSync } from 'bcryptjs'
 import { Factory } from 'rosie'
 import { getDriver, getNeode } from './neo4j'
+import CONFIG from '../config/index.js'
 
 const neode = getNeode()
 
+const uniqueImageUrl = imageUrl => {
+  const newUrl = new URL(imageUrl, CONFIG.CLIENT_URI)
+  newUrl.search = `random=${uuid()}` // make it unique
```


```suggestion
  newUrl.search = `random=${uuid()}`
```
if you push up again, I guess you can erase this comment, it's clear why you are doing this, oder?